projects/fodl/index.js

```diff
@@ -90,6 +99,5 @@ module.exports = {
   ethereum: {
     tvl: tvl,
   },
-  tvl,
   pool2: ethPool2,
```

Comment:
would be better to put pool2 inside ethereum, that way we know in which network pool2 is in